chargebee_byte/client.py

```diff
@@ -13,3 +14,14 @@ def get_subscriptions(self, parameters=None):
         response = requests.get(self.api_url + request.path, auth=self.auth, params=request.data)
         response.raise_for_status()
         return response.json()
+
+    def get_all_subscriptions(self, parameters=None):
```

Comment:
Why not just change `get_subscriptions` to do this? I don't see why you would ever want just the paginated data back.

Reply:
You never know, don't think it can hurt to allow more variation? Maybe some people want to have some pagination somewhere in that case I think it would be easier to just use the pagination provided by chargebee.

Reply:
In that case it might be worthwhile to reflect this difference in the method names. Right now both names are semantically equivalent, imho, while they have distinct functionality. Perhaps something like `get_subscriptions_page` and `get_all_subscriptions`?

Reply:
@thomasbrockmeier I've renamed the `get_subscriptions` function